packages/create-app/templates/default-app/.dockerignore

```diff
@@ -3,3 +3,4 @@ node_modules
 packages
 !packages/backend/dist
 plugins
+app-config.local.yaml
```

Comment:
Think it could make sense to have this match the `.gitignore`, so `*.local.yaml`?

Reply:
Ah yeah, might as well.